internal/hcsoci/pnp.go

```diff
@@ -0,0 +1,58 @@
+// +build windows
+
+package hcsoci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/Microsoft/hcsshim/internal/shimdiag"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+	"github.com/pkg/errors"
+)
+
+const uvmPnpExePath = "C:\\Windows\\System32\\pnputil.exe"
+
+// createPnPInstallDriverCommand creates a pnputil command to add and install drivers
+// present in `driverUVMPath` and all subdirectories.
+func createPnPInstallDriverCommand(driverUVMPath string) []string {
+	dirFormatted := fmt.Sprintf("%s/*.inf", driverUVMPath)
+	args := []string{
+		"cmd",
```

Comment:
It seems simpler to just do multiple `ExecProcessRequest`s in the UVM, rather than a single request with all of the command lines concatenated. Is there a particular reason we do it this way?

Reply:
I was just trying to reduce the total number of exec calls we make into the UVM. I can do some explicit perf testing with both to see the impact.

Reply:
Unless there is a large performance impact, I think we should just use multiple invocations.

Reply:
Done. But still testing this change.

Reply:
Verified